service-two/pom.xml

```diff
@@ -1,85 +1,93 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <parent>
-        <groupId>org.springframework.boot</groupId>
-        <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.3.5.RELEASE</version>
-        <relativePath/> <!-- lookup parent from repository -->
-    </parent>
-    <modelVersion>4.0.0</modelVersion>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.4.0</version>
+		<relativePath/> <!-- lookup parent from repository -->
+	</parent>
+	<groupId>com.okta.dev</groupId>
+	<artifactId>service-two</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<name>service-two</name>
+	<description>Microservice Two</description>
 
-    <groupId>com.okta.dev.springcloudconfig</groupId>
-    <artifactId>service-two</artifactId>
-    <name>service-two</name>
-    <description>Service Two</description>
+	<properties>
+		<java.version>11</java.version>
+		<spring-cloud.version>2020.0.0-M6</spring-cloud.version>
```

Comment:
Is the new version necessary to work with Spring Boot 2.4?

Reply:
This is what start.spring.io generates. It might be.